certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificate/persistence/repository/CertificateEntitySpecificationFactory.java

```diff
@@ -87,6 +90,25 @@ public Specification<CertificateEntity> create(CertificatesRequest certificatesR
         notPlacerholderCertificate()
     );
 
+    if (certificatesRequest.signedFrom() != null) {
+      specification = specification.and(
+          signedEqualsAndGreaterThan(certificatesRequest.signedFrom())
+      );
+    }
+
+    if (certificatesRequest.signedTo() != null) {
+      specification = specification.and(
+          signedEqualsAndLesserThan(certificatesRequest.signedTo())
+      );
+    }
+
+    if (certificatesRequest.issuedByUnitIds() != null && !certificatesRequest.issuedByUnitIds()
```

Comment:
See comment on CertificateRequest. Instead of adding this modify the current one to manage a list.